modules/33acrossBidAdapter.js

```diff
@@ -5,10 +5,20 @@ import * as utils from '../src/utils.js';
 const BIDDER_CODE = '33across';
 const END_POINT = 'https://ssc.33across.com/api/v1/hb';
 const SYNC_ENDPOINT = 'https://ssc-cms.33across.com/ps/?m=xch&rt=html&ru=deb';
-const MEDIA_TYPE = 'banner';
+const MEDIA_TYPE = {
```

Comment:
if this is for our internal usage, we can leave it like this, otherwise we can use the constant from prebid js

mediaType: https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L5
videoType: https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L9